src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -180,13 +108,17 @@ + (NSString *)hostNameOverride {
   return nil;
 }
 
++ (BOOL)useCronet {
+  return NO;
+}
+
 + (void)setUp {
   NSLog(@"InteropTest Started, class: %@", [[self class] description]);
 #ifdef GRPC_COMPILE_WITH_CRONET
-  // Cronet setup
-  [Cronet setHttp2Enabled:YES];
-  [Cronet start];
-  [GRPCCall useCronetWithEngine:[Cronet getGlobalEngine]];
+  enableCronet();
+  if ([self useCronet]) {
+    [GRPCCall useCronetWithEngine:[Cronet getGlobalEngine]];
```

Comment:
You are right. However, the fact here is, `enableCronet()` sets up the global Cronet engine; the if statement asks gRPC v1 API to use Cronet. We do need to set up the global engine always, because the v2 API also needs Cronet to be compiled.

I think the right fix is to change the name `enableCronet` to something like `configureCronet` to make it less confusing.